sql/dd/types/init_mode.h

```diff
@@ -0,0 +1,37 @@
+/*
+   Copyright (c) 2000, 2018, Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2019 Percona LLC and/or its affiliates. All rights reserved.
```

Comment:
Why Percona copyright? This is just Oracle code in different file

Reply:
But the file is ours, so there was a change. Laurynas asked me to put Oracle copyrights here, next to Percona one.

Reply:
@laurynas-biveinis Are you sure with Percona copyright here? There is nothing new by percona here.

Reply:
Our new files should carry our copyright. Let's keep both copyrights and let someone better versed in these matters to change it later if needed

Reply:
Ok